packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -531,8 +537,7 @@ export function getValidatorApi(
     slot: Slot,
     randaoReveal: BLSSignature,
     graffiti?: string,
-    // TODO deneb: skip randao verification
-    _skipRandaoVerification?: boolean,
+    skipRandaoVerification?: boolean,
```

Comment:
did you check if there is something in the spec that requires us to verify the randao reveal if this flag is omitted? If not we might wanna keep current behavior on set this value explicitly to `true` here as default and only validate if flag is explicitly set to `false` by caller


the flag name itself suggest that default behavior should be to verify, but we should check how expensive `processRandao` is before changing default behavior

Reply:
When I was trying to understand this issue, I searched a bit, but there was little data about it, perhaps merely the link in the linked issue. The Beacon APIs description of the field states no default value for it.  
It seems Lighthouse, for example, are defaulting to `false` for it and verify the reveal by default if the query param is not provided:  
https://github.com/sigp/lighthouse/blob/999b04517e35f8fce78525132d0b8e0a4816f31e/common/eth2/src/types.rs#L761-L767
As you can see, the default is `No`, probably if the `Option<String>` is empty. By the way, I don't know the _cost_ of defaulting to verification.

Reply:
the flag name itself `skip_randao_verification` kinda suggest that default should be to verify but I don't think we should change or default behavior but rather if a caller explicilty passes `skip_randao_verification=false` then we should validate it